src/hdx_cli/cli_interface/common/undecorated_click_commands.py

```diff
@@ -382,6 +387,7 @@ def basic_delete(profile, resource_path, resource_name: str):
             break
     if not url:
         return False
+    url += '?force_operation=true' if force_operation else ''
```

Comment:
Even though this is a simple case, it's best to use either the urllib or requests libraries to create query parameters.

examples with urllib
```python
>>> import urllib
>>> params = {"force_operation": True}
>>> urllib.parse.urlencode(params)
'force_operation=True'
```
example with requests
```python
>>> import requests
>>> url = "https://hydrolix.com/"
>>> params = {"force_operation": True}
>>> response = requests.get(url, params=params)
>>> print(response.url)
https://hydrolix.io/?force_operation=True
```